lib/y2_mm_common.pm

```diff
@@ -9,6 +14,15 @@ use x11utils 'turn_off_gnome_screensaver';
 
 our @EXPORT = qw(prepare_xterm_and_setup_static_network);
 
+=head2 prepare_xterm_and_setup_static_network
+
+ prepare_xterm_and_setup_static_network();
+
+Use C<x11_start_program('xterm')> to open a xterm. 
+Then setup a static network by C<configure_static_network($args{ip})>.
+C<$args> can be {message} or {ip}.
```

Comment:
That's not 100% correct, as here we accept hash and not hash reference. So let's write `%args` and then list possible  keys.

Reply:
yes, updated now. thanks!